tests/base/develop_pipelines.py

```diff
@@ -47,10 +47,14 @@ def run_model_test(trainer_options, model, on_gpu: bool = True, version=None, wi
         trainer_options.update(checkpoint_callback=True)
 
     trainer = Trainer(**trainer_options)
+    initial_values = torch.tensor([torch.sum(torch.abs(x)) for x in model.parameters()])
     result = trainer.fit(model)
+    post_train_values = torch.tensor([torch.sum(torch.abs(x)) for x in model.parameters()])
 
     # correct result and ok accuracy
     assert result == 1, 'trainer failed'
```

Comment:
can you remove the comment? the assertion does not actually do what the comment says :)